### DISCUSSION TITLE
Conditionally Rendered Components Causing Funny Business with the Browser Address Bar

### DISCUSSION THREAD

**Discussion by unohuim** [livewire/livewire#3074] [7. Bugs (archived)]
### Description
I have a full page livewire component, Client, that will show a number of smaller livewire components like services, series, contacts, pets, invoices, etc.  Initially I was stuffing them inside accordion views.  That worked fine, BUT, I didn't want all of those components loading at the same time negatively impacting TTFB.  So, I decided that a mini SPA would work well.  So now I have a div that renders content specific to the user click, which emits an event that eventually changes a string ($content) so that the blade file will know which livewire component to render.

When I render a livewire component conditionally, the browser address bar gets changed to {app.domain}/livewire/messaging/{current-component-name} and breaks my app.  It basically adds livewire/messaging/ between the domain and /{current-component-name}.  If I remove the conditioning, then everything works fine.

The code below results in the browser address bar being set to,
https://app_woofs.test/livewire/message/client

### Exact steps to reproduce
Just include the code below into my view

### Stripped-down, copy-pastable code snippets

###produces error as mentioned
@if( $content==="client-series" )
    <livewire:client-series :client="$thisClient" />
@endif

vs

###works perfectly well as advertised
<livewire:client-series :client="$thisClient" />

**Context**
- Livewire version: [e.g. 2.2.3]
- Laravel version: [e.g. 8.3.1]
- Alpine version: [e.g. 2.3.0]
- Browser: [e.g. Chrome, Safari, those are my only two - both behave the same here]

Thanks in advance!
Rob

**Comment by cybsom** [comment]
Faced the same problem.

If you add "**use WithPagination;**" to the parent component, everything will work.

It turns out some kind of problem with queryString.
